docs/details/types.md

```diff
@@ -200,25 +200,26 @@ type.isAnonymous();
 type.isAny();
```

Comment:
I ordered them alphabetically and added `isTemplateLiteral` to the list, that's why you see multiple changes here 🙂